### ISSUE TITLE
So many requirements?

### ISSUE THREAD

**Issue by Dumbris** [cmooredev/RepoReader#3]
I have noticed that there are a lot of libraries in requirements.txt, maybe you need to clean it up? 🤔

**Comment by repollo** [comment]
Yes, apparently OP just `pip freeze > requirements.txt` since you only require these packages for this project to run,

```python
python-dotenv
nltk
langchain
scikit-learn
rank-bm25
```
